gui/src/components/commons/Dropdown.tsx

```diff
@@ -37,10 +41,39 @@ export function Dropdown({
       }
     }
 
-    document.addEventListener('wheel', onWheelEvent, { passive: true });
+    function onTouchEvent(event: TouchEvent) {
+      // Check if we touch scroll outside of the dropdown
+      if (
+        isOpen &&
+        !document
+          .querySelector('div.dropdown-scroll')
+          ?.contains(event.target as any)
+      ) {
+        setOpen(false);
+      }
+    }
+
+    function onClick(event: MouseEvent) {
+      if (
+        isOpen &&
+        !document
+          .querySelector('div.dropdown-scroll')
+          ?.contains(event.target as any) &&
+        !document.querySelector('div.dropdown')?.contains(event.target as any)
+      ) {
```

Comment:
```suggestion
      const isInDropdownScroll = document
        .querySelector('div.dropdown-scroll')
        ?.contains(event.target as any);
      const isInDropdown = document.querySelector('div.dropdown')?.contains(event.target as any);
      if (isOpen && !isInDropdownScroll && !isInDropdown) {
```

Reply:
Any is used because typescript definitions for event.target is wrong